demo/www/lib/ngCordova/dist/ng-cordova.js

```diff
@@ -5007,7 +5011,7 @@ angular.module("ngCordova.plugins.oauth", ["ngCordova.plugins.oauthUtility"])
             var deferred = $q.defer();
             if(window.cordova) {
                 var cordovaMetadata = cordova.require("cordova/plugin_list").metadata;
-                if(cordovaMetadata.hasOwnProperty("org.apache.cordova.inappbrowser") === true) {
+                if(cordovaMetadata.hasOwnProperty("org.apache.cordova.inappbrowser") === true || cordovaMetadata.hasOwnProperty("cordova-plugin-inappbrowser") === true || cordovaMetadata.hasOwnProperty("cordova-plugin-inappbrowser") === true) {
```

Comment:
@JoshAntBrown I also made a Pull Request to `ng-cordova-oauth` project doing that, if you want, take a look: https://github.com/nraboy/ng-cordova-oauth/pull/83